README.md

```diff
@@ -18,6 +18,12 @@ Currently it provides you with: a few utilities, images/sprites/sprite sheets, a
 
 You can use whatever ones you want, as they are all somewhat independent.
 
+## Donate
+
+Donate to support Flame's development:
+
+[![Donate](https://img.shields.io/badge/Donate-PayPal-green.svg)](https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=VJS3922363XNU&currency_code=USD&source=url)
+
```

Comment:
Good idea! Let's create a Fireslime paypal account, though heheheh

Reply:
Good!! :)

Reply:
Maybe this is a better idea: https://www.paypal.com/us/webapps/mpp/money-collection#

What do you think @luanpotter @feroult ?

Reply:
Looks cool, but does it need a fixed target amount? Or can it stay open forever?

Reply:
@luanpotter I think it should be open. (any cons?)

@erickzanardo never used that... but if it works....  :)